odoo/addons/base/models/ir_http.py

```diff
@@ -464,12 +467,19 @@ def binary_content(self, xmlid=None, model='ir.attachment', id=None, field='raw'
         :param str default_mimetype: default mintype if no mintype found
         :param str access_token: optional token for unauthenticated access
                                  only available  for ir.attachment
+        :param str allow_xaccel: optional argument to bypass xaccel NGINX serving
+                                and fetch raw data instead of only a header
         :returns: (status, headers, content)
         """
         record, status = self._get_record_and_check(xmlid=xmlid, model=model, id=id, field=field, access_token=access_token)
+        xaccel_header = False
```

Comment:
This typing makes no sense, why does this get initialised to `False`?

Reply:
Yes, None would make more sense I suppose. Will change it to that.